commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -111,15 +110,14 @@ class CreateReturnForCommercetoolsOrder(PipelineStep):
     def run_filter(
             self,
             active_order_management_system,
-            order_number,
+            order_id,
             order_line_id
     ):  # pylint: disable=arguments-differ
         """
         Execute a filter with the signature specified.
         Args:
             active_order_management_system: The Active Order System
             order_number: Order number (for now this is an order.id, but this should change in the future)
```

Comment:
Need to update this argument in the docstring.

```suggestion
            order_id: Order ID
```

Reply:
I think I misunderstood. The intention was to eventually replace the order.id with an actual order number, not to replace the order number argument with order id.